### ISSUE TITLE
/usr/local/bin/wpscan:23:in `<main>'

### ISSUE THREAD

**Issue by hktalent** [wpscanteam/wpscan#1328]
gem install wpscan

in Kali Linux

ruby -v
ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-linux-gnu]

```
[i] Updating the Database ...

Scan Aborted: No such file or directory @ rb_sysopen - /root/.wpscan/db/user-agents.txt
Trace: /var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/browser/options.rb:65:in `initialize'
/var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/browser/options.rb:65:in `open'
/var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/browser/options.rb:65:in `user_agents'
/var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/browser/options.rb:56:in `user_agent'
/var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/browser.rb:51:in `default_request_params'
/var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/browser.rb:79:in `request_params'
/var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/browser.rb:38:in `forge_request'
/var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/browser/actions.rb:12:in `forge_request'
/var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/browser/actions.rb:20:in `get'
/var/lib/gems/2.5.0/gems/wpscan-3.5.0/lib/wpscan/db/updater.rb:84:in `remote_file_checksum'
/var/lib/gems/2.5.0/gems/wpscan-3.5.0/lib/wpscan/db/updater.rb:139:in `block in update'
/var/lib/gems/2.5.0/gems/wpscan-3.5.0/lib/wpscan/db/updater.rb:137:in `each'
/var/lib/gems/2.5.0/gems/wpscan-3.5.0/lib/wpscan/db/updater.rb:137:in `update'
/var/lib/gems/2.5.0/gems/wpscan-3.5.0/app/controllers/core.rb:47:in `update_db'
/var/lib/gems/2.5.0/gems/wpscan-3.5.0/app/controllers/core.rb:57:in `before_scan'
/var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/controllers.rb:45:in `each'
/var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/controllers.rb:45:in `block in run'
/usr/lib/ruby/2.5.0/timeout.rb:76:in `timeout'
/var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/controllers.rb:44:in `run'
/var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/scan.rb:24:in `run'
/var/lib/gems/2.5.0/gems/wpscan-3.5.0/bin/wpscan:16:in `block in <top (required)>'
/var/lib/gems/2.5.0/gems/cms_scanner-0.0.43.2/lib/cms_scanner/scan.rb:15:in `initialize'
/var/lib/gems/2.5.0/gems/wpscan-3.5.0/bin/wpscan:6:in `new'
/var/lib/gems/2.5.0/gems/wpscan-3.5.0/bin/wpscan:6:in `<top (required)>'
/usr/local/bin/wpscan:23:in `load'
/usr/local/bin/wpscan:23:in `<main>'
```

**Comment by erwanlr** [comment]
Was not able to reproduce on the latest Kali.

Things I can think of is that there is one of the files below (pwd is the current directory):
```
~/.wpscan/cli_options.json
~/.wpscan/cli_options.yml
pwd/.wpscan/cli_options.json
pwd/.wpscan/cli_options.yml
```
containing the --user-agent-list option set.

Furthermore, manually installing gems on Kali is NOT recommended as it might screw up things.

**Comment by erwanlr** [comment]
Another idea, if you had the `--random-user-agent` while trying to do the first update, remove the option and try again

**Comment by erwanlr** [comment]
Fixed in 3.5.1 :)
